tests/unit/test_cubeset.py

```diff
@@ -0,0 +1,176 @@
+# encoding: utf-8
+
+"""Unit test suite for `cr.cube.cube.Cubeset` class."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+import numpy as np
+
+from cr.cube.cube import CubeSet, Cube
+from cr.cube.cubepart import _Slice, _Strand, _Nub
+from cr.cube.enum import DIMENSION_TYPE as DT
+
+from ..unitutil import instance_mock, property_mock
+
+
+class DescribeCrunchCubeSet(object):
+    def it_knows_whether_it_has_means(self, has_means_fixture, _cubes_prop_, cube_):
+        first_cube_has_means, expected_value = has_means_fixture
+        cube_.has_means = first_cube_has_means
+        _cubes_prop_.return_value = (cube_,)
+        cube_set = CubeSet(None, None, None, None)
+
+        has_means = cube_set.has_means
+
+        assert has_means == expected_value
+
+    def it_knows_whether_it_has_weighted_counts(
+        self, has_weighted_counts_fixture, _cubes_prop_, cube_
+    ):
+        first_cube_has_w_counts, expected_value = has_weighted_counts_fixture
+        cube_.is_weighted = first_cube_has_w_counts
+        _cubes_prop_.return_value = (cube_,)
+        cube_set = CubeSet(None, None, None, None)
+
+        is_weighted = cube_set.has_weighted_counts
```

Comment:
In general, name the result variable exactly as the method or property under test (without any leading underscore). This reduces the concepts to be understood and improves speed-readability. It also makes writing faster because you don't have to reason about what a good name would be. So in this case:

```python
has_weighted_counts = cube_set.has_weighted_counts

assert has_weighted_counts == expected_value
```